lms/static/sass/shared/_course_object.scss

```diff
@@ -7,33 +7,44 @@
       width: flex-grid(4);
       @include margin-right(flex-gutter());
       float: left;
-
-      &:nth-child(3n+3) {
-        @include margin-right(0);
-      }
     }
 
     .courses-listing {
+      @include outer-container;
       @include clearfix();
       margin: 0;
       padding: 0;
       list-style: none;
 
       .courses-listing-item {
-        width: flex-grid(4);
-        @include margin-right(flex-gutter());
-        @include float(left);
+        height: 360px;
```

Comment:
Why the fixed height here? Is it to reinforce all course cards have the same height? If so, I'd recommend doing:

1. setting  a ``max-height`` instead
2. using the $baseline variable (our base unit of vertical/horizontal spacing) to help calculate this value (e.g. ``max-height: ($baseline*18)``)
3. creating Sass variables that you define at the top of this file, including one that is for ``$course-card-height`` and referencing it here.